varats-core/varats/base/configuration.py

```diff
@@ -36,13 +38,13 @@ def __ne__(self, other: tp.Any) -> bool:
         return not self == other
 
 
-class Configuration():
+class Configuration:
     """Represents a specific configuration of a project, e.g., encapsulating
     static and run-time options on how the project should be build."""
 
     @staticmethod
     @abc.abstractmethod
-    def create_configuration_from_str(config_str: str) -> 'Configuration':
+    def create_configuration_from_str(config_str: str) -> "Configuration":
```

Comment:
```suggestion
    def create_configuration_from_str(config_str: str) -> 'Configuration':
```
We normally use single quotes for types.